server/src/infra/repositories/search.repository.ts

```diff
@@ -220,6 +221,96 @@ export class SearchRepository implements ISearchRepository {
       .getMany();
   }
 
+  @GenerateSql({ params: [[DummyValue.UUID]] })
+  async getAssetsByCity(userIds: string[]): Promise<AssetEntity[]> {
```

Comment:
Man this is ugly.

Reply:
It was too much for the query builder haha